specification/graphrbac/data-plane/stable/1.6/graphrbac.json

```diff
@@ -1628,13 +1628,13 @@
       },
       "post" : {
        "tags": [
-         "OAuth2Permissions_post"
+         "OAuth2Permissions_Grants"
```

Comment:
Making this plural suggests a read operation to me. I'd recommend "OAuth2Permissions_Grant" or even "OAuth2Permissions_CreateOrUpdate"

Reply:
For a POST action I would avoid "create_or_update", I go for "grant" :)